client/src/main/java/edu/snu/nemo/client/JobLauncher.java

```diff
@@ -78,16 +84,14 @@ private JobLauncher() {
    * @throws Exception exception on the way.
    */
   public static void main(final String[] args) throws Exception {
-    final DriverRPCServer driverRPCServer = new DriverRPCServer();
+    driverRPCServer = new DriverRPCServer();
+
     // Registers actions for launching the DAG.
     driverRPCServer
         .registerHandler(ControlMessage.DriverToClientMessageType.DriverStarted, event -> { })
-        .registerHandler(ControlMessage.DriverToClientMessageType.ResourceReady, event ->
-          driverRPCServer.send(ControlMessage.ClientToDriverMessage.newBuilder()
-              .setType(ControlMessage.ClientToDriverMessageType.LaunchDAG)
-              .setLaunchDAG(ControlMessage.LaunchDAGMessage.newBuilder().setDag(serializedDAG).build())
-              .build()))
-        .registerHandler(ControlMessage.DriverToClientMessageType.DataCollected, message -> collectedData.addAll(
+        .registerHandler(ControlMessage.DriverToClientMessageType.ResourceReady, event -> driverReadyLatch.countDown())
```

Comment:
`ResourceReady` vs `driverReady`. This is somewhat confusing.

Reply:
I'll rename it to DriverReady. I tried to follow what was already there, but I think it would be better to make the change.